views/admin.functions.php

```diff
@@ -246,7 +246,7 @@ function sailthru_account_settings() {
  */
 function sailthru_status() {
 
-	// default to false
+	// this will default to false
```

Comment:
**suggestion**: Remove this comment instead of updating it. The comment is not adding clarity to the code.